grails-app/domain/com/unifina/domain/security/IntegrationKey.groovy

```diff
@@ -20,6 +20,7 @@ class IntegrationKey implements Serializable {
 	static mapping = {
 		id generator: IdGenerator.name // Note: doesn't apply in unit tests
 		json type: 'text'
+		user lazy: false
```

Comment:
this seemed like the only "transitive" domain object contained in IntegrationKey

Reply:
(which could be lazy-loaded, that is; or eagerly hopefully from now on)